### ISSUE TITLE
Support transitions for changing data

### ISSUE THREAD

**Issue by tanc** [benkeen/d3pie#51]
More of a support request at the moment but can this library support transitions between data states? For example growing and shrinking the size of the pie based on updated data. From what I understand only redraws are supported at the moment.

**Comment by benkeen** [comment]
Yeah, only redraws work right now. I agree, this would be a very desirable feature.

**Comment by billgarlic** [comment]
I was wondering if there was any plan to implement this feature.  I would love to use it as well.  For the moment, redraw is pretty efficient, just not as visually desirable to show changing data levels.

Otherwise, really great pie chart generator.
